docs/installation/windows.md

```diff
@@ -9,122 +9,296 @@ parent = "smn_engine"
+
+    ![Which version](/installation/images/win_ver.png)
+    
+    If you aren't using a supported version, you could consider upgrading your
+    operating system.
+
+2. Make sure your Windows system supports Hardware Virtualization Technology and that virtualization is enabled.
+
+    #### For Windows 8 or 8.1
+
+	  Choose **Start > Task Manager** and navigate to the **Performance** tab.          
+	  Under **CPU** you should see the following:
+
+      ![Release page](/installation/images/virtualization.png)
+    
+    If virtualization is not enabled on your system, follow the manufacturer's instructions for enabling it.
+    
+    ### For Windows 7 
+    
+	  Run the <a
+	  href="http://www.microsoft.com/en-us/download/details.aspx?id=592"
+	  target="_blank"> Microsoft® Hardware-Assisted Virtualization Detection
+	  Tool</a> and follow the on-screen instructions.*
+
+Your processor needs to support hardware virtualization.
+### Learn the key concepts before installing
+
+In a Docker installation on Linux, your machine is both the localhost and the
+Docker host. In networking, localhost means your computer. The Docker host is
+the machine on which the containers run.
+
+On a typical Linux installation, the Docker client, the Docker daemon, and any
+containers run directly on your localhost. This means you can address ports on a
+Docker container using standard localhost addressing such as `localhost:8000` or
+`0.0.0.0:8376`.
+
+![Linux Architecture Diagram](/installation/images/linux_docker_host.svg)
+
+In an Windows installation, the `docker` daemon is running inside a Linux virtual
+machine. You use the Windows Docker client to talk to the Docker host VM. Your
+Docker containers run inside this host. 
 
 ![Windows Architecture Diagram](/installation/images/win_docker_host.svg)
 
-## Demonstration
+In Windows, the Docker host address is the address of the Linux VM. When you
+start the VM with `docker-machine` it is assigned an IP address. When you start
+a container, the ports on a container map to ports on the VM. To see this in
+practice, work through the exercises on this page.
+
+>**Note**: Until the Docker engine for Windows is developed, you can launch only
+>Linux containers from your Windows machine.
+
+### Installation
+
+If you have Virtual Box running, you must shut it down before running the
+installer. 
+
+1. Go to the [Docker Toolbox](https://docker-toolbox.azurewebsites.net/) release page.
+
+4. Click "DOWNLOAD DOCKER TOOLBOX."
+
+3. Install Docker Toolbox by double-clicking the installer.
+
+    The installer launches the "Setup - Docker Toolbox" dialog.
+    
+    ![Install Docker Toolbox](/installation/images/win-welcome.png)
+
+4. Press "Next" to install the toolbox.
+
+    The installer presents you with options to customize the standard
+    installation. By default, the standard Docker Toolbox installation:
+  
+    * installs executables for the Docker tools in `C:\Program Files\Docker Toolbox` 
```

Comment:
Nit: could be `C:\Program Files (x86)\Docker Toolbox`, not sure, may be worth checking 

(I hate those paths, LOL )